Plotting/individual_plots.py

```diff
@@ -215,9 +198,12 @@ def indiv_derived_props_plots(fitspath, restrictMTO = False, revised = False, er
         ax11.errorbar(bin_logM[bin_detect], bin_metal[bin_detect], yerr = err_dict['12+log(O/H)_lowhigh_error'], fmt = '.')
     
     #Fit bin detections and plot relation
-    o11, o21, fail = curve_fitting(bin_logM[bin_detect], bin_metal[bin_detect], restrict_MTO = restrictMTO)   
+    o11, o21, fail = curve_fitting(bin_logM[bin_detect], bin_metal[bin_detect], restrict_MTO = restrictMTO)
     if not fail:
-        ax11.plot(mass_range, mass_metal_fit(mass_range, *o11), alpha = 0.5, color = 'red', label = 'Our Fit')
+        if restrictMTO:
+            ax11.plot(mass_range, mass_metal_fit_constMTO(mass_range, *o11), alpha = 0.5, color = 'red', label = 'Our Fit')
+        else:
+            ax11.plot(mass_range, mass_metal_fit(mass_range, *o11), alpha = 0.5, color = 'red', label = 'Our Fit')
```

Comment:
You could actually define `o11` to include the MTO fixed value and therefore just need `mass_metal_fit`.  This, I believe, would need to be done in `curve_fitting`